pages/linux/retroarch.md

```diff
@@ -0,0 +1,29 @@
+# retroarch
+
+> RetroArch is a frontend for emulators, game engines and media players.
+> It is the reference implementation of libretro API.
+> More information: <https://github.com/libretro/RetroArch>.
+
+- Start `retroarch` in menu mode:
```

Comment:
```suggestion
- Start `retroarch` in the menu mode:
```

Are there several menu modes? If so then prefer `a`.

Reply:
By default it goes into the menu, menu mode means it opens up the menu in retroarch. I can also open a game ROM directly without getting into menu mode. So, I will go with the current suggestion then.